docs/site_pipeline.rst

```diff
@@ -572,6 +572,115 @@ API
 .. autoclass:: sotodlib.site_pipeline.monitor.Monitor
     :members:
 
+make-position-match
```

Comment:
Move this up, before make-source-flags (i.e. alphabetical in the other elements).  Right now it's rendering inside "QDS Monitor".